### PR TITLE
adding nav tag templates

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@saeedmb8 ready for review

File: gem/templates/springster/core/main.html [hunk 1]
```diff
@@ -3,7 +3,17 @@
 
 {% block content %}
     {% bannerpages %}
-    {% topic_of_the_day %}
-    {% latest_listing_homepage num_count=6 %}
-    {% section_listing_homepage %}
+
+    {% if not settings.core.SiteSettings.enable_tag_navigation %}
+      {% topic_of_the_day %}
```
Comment (saeedmb8) [inline gem/templates/springster/core/main.html]: Gem doesn't use topic of day

Comment (codiebeulaine) [inline gem/templates/springster/core/main.html]: okay

**Comment by saeedmb8** [comment]
👍

**Pull request merged**